packages/bz3.rb

```diff
@@ -0,0 +1,51 @@
+require 'package'
+
+class Bz3 < Package
+  description 'bzip3 is a better and stronger spiritual successor to bzip2.'
+  homepage 'https://github.com/kspalaiologos/bzip3'
+  @_ver = '1.1.6'
+  version @_ver
+  license 'LGPL-3'
+  compatibility 'all'
+  source_url 'https://github.com/kspalaiologos/bzip3.git'
+  git_hashtag @_ver
+
+  depends_on 'glibc' # R
+
+  def self.patch
+    # Remove autoreconf from bootstrap.sh: we'll run it ourselves
+    system "sed -i 's:^autoreconf:#autoreconf:g' bootstrap.sh"
+
+    # Replace /bin/sh shebangs with /usr/bin/env sh ones
+    ["bz3cat", "bz3grep", "bz3less", "bz3more", "unbzip3"].each do |file|
+      system "sed -i 's:^#!/bin/sh:#!/usr/bin/env sh:' #{file}"
+    end
+
+    # Add support for most
+    FileUtils.cp 'bz3less', 'bz3most'
+    system "sed -i 's:less:most:g' bz3most"
+    system "sed -i 's: bz3less : bz3less bz3most :g' Makefile.am"
+
+    FileUtils.cp 'bz3less.1', 'bz3most.1'
+    system "sed -i 's:less:most:g' bz3most.1"
+    system "sed -i 's: bz3less.1 : bz3less.1 bz3most.1 :g' Makefile.am"
```

Comment:
Good call!  Would it make sense to add most support instead of removing less support?